patches/chrome-BUILD.gn.patch

```diff
@@ -110,3 +110,12 @@ index a0db274a53134fb1f1085185fc1030f6bc78917b..9f5ac285bdf4dd9d8c6285df70c744dd
    ]
  }
  
+@@ -1612,6 +1623,8 @@ if (!is_android) {
+     if (enable_resource_allowlist_generation) {
+       repack_allowlist = _chrome_resource_allowlist
+       deps = [ ":resource_allowlist" ]
++      repack_allowlist = brave_allowlist_file
```

Comment:
this patch still needs to be fixed to be on one line

Reply:
done